### DISCUSSION TITLE
[Kavita] Add an option to jump to next chapter/volume in PDF

### DISCUSSION THREAD

**Discussion by Javitsao** [Kareadita/Kavita#3144] [Ideas]
### Idea Description

When reading PDF comics, after finishing a chapter and wanting to read the next one, I have to press 'x' to close the current chapter, then scroll down from the top to the next chapter, and press 'Read' to continue.

### Idea Category

User Experience

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by Fethbita** [comment]
I would also be happy about this feature. Currently it takes too much time to go back and continue reading the next pdf, and I have to change the zoom level to something suitable every time I open the next chapter.
